packages/ingest/test/test-FtpProviderClientFailure.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const test = require('ava');
+const JSFtp = require('jsftp');
+
+const FtpProviderClient = require('../FtpProviderClient');
+
+const executeRawFtpCommand = async (cmd) => {
+  const jsftp = new JSFtp({
+    host: '127.0.0.1',
+    port: 21,
+    user: 'testuser',
+    pass: 'testpass'
+  });
+  return new Promise((resolve, reject) => {
+    jsftp.raw(cmd, (err, data) => {
+      if (err) reject(err);
+      resolve(data);
+    });
+  });
+};
+
+test.before(async () => {
+  // await when fixed
+  executeRawFtpCommand('chmod 400 -R forbidden/file.txt');
```

Comment:
I fear that the answer is obvious, but why aren't we `await`ing?

Reply:
This was the only way to allow this file to satisfy both `ava` and `eslint`. If we `await` this command throws an error because our server's `vsftpd.conf` does not have `chmod_enabled=true`.